MatrixSDK/MXSession.m

```diff
@@ -1544,18 +1544,7 @@ - (void)handleServerSyncError:(NSError*)error forRequestWithServerTimeout:(NSUIn
         int32_t code = (int32_t)error.code;
 
         if ([error.domain isEqualToString:NSURLErrorDomain]
-            && code == kCFURLErrorCancelled)
```

Comment:
When we remove this if check, session goes into `homeServerNotReachable` state when we pull-to-refresh, let's just keep the check and do nothing (we could just keep the log line) inside the block.

Reply:
Makes sense, added the log back